robot/Cumulus/tests/browser/recurring_donations/check_recurring_auto_naming.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Run Keywords
+...             Open Test Browser
+...             Setup Variables
+Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+Setup Variables
+    ${ns} =                      Get NPSP Namespace Prefix
+    Set Suite Variable           ${ns}
+
+*** Test Cases ***
+
+Check Recurring Donation Autoname
+    [Documentation]              This test verifies that the Autonaming setting for Recurring Donations is working properly.
+
+    #Enable Autonaming Setting
+    @{recurringsettings} =       Salesforce Query                           npe03__Recurring_Donations_Settings__c
+    ...                          select=Id,EnableAutomaticNaming__c
```

Comment:
you forgot namespace for the EnableAutomaticNaming__c here

Reply:
@skristem thanks made that change!